commit 39583ff118920284de516d262979960e7159bcfc
Author: Jose Dapena Paz <jdapena@igalia.com>
Date:   Thu Sep 19 15:44:36 2024 +0000

    IWYU: missing include for usage of std::stack in skia_image_decoder_base.cc
    
    Bug: 41455655
    Change-Id: Id9c09f951dacc34d46aadc4badc6b3fd250ad38b
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5872492
    Reviewed-by: Peter Kasting <pkasting@chromium.org>
    Commit-Queue: Peter Kasting <pkasting@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1357626}

diff --git a/third_party/blink/renderer/platform/image-decoders/skia/skia_image_decoder_base.cc b/third_party/blink/renderer/platform/image-decoders/skia/skia_image_decoder_base.cc
index 1d002cc258ae8..84fd71c1e4613 100644
--- a/third_party/blink/renderer/platform/image-decoders/skia/skia_image_decoder_base.cc
+++ b/third_party/blink/renderer/platform/image-decoders/skia/skia_image_decoder_base.cc
@@ -5,6 +5,7 @@
 #include "third_party/blink/renderer/platform/image-decoders/skia/skia_image_decoder_base.h"
 
 #include <limits>
+#include <stack>
 
 #include "third_party/blink/renderer/platform/image-decoders/segment_stream.h"
 #include "third_party/skia/include/codec/SkCodec.h"
